File: existant/gestFormationWeb/gestion_formation_web/dto/DtoContact.cs
using System;
using gestion_formation_web.dao;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace gestion_formation_web.dto
{
    public class DtoContact
    {
        public static contact Get(int idContact)
        {
            return DaoContact.Get(idContact);
        }
    }
}

File: Gestionnaire de formation/src/main/java/fr/esic/solutec/dto/DtoContact.java
package fr.esic.solutec.dto;

import java.util.List;


import fr.esic.solutec.dao.DaoContact;
import fr.esic.solutec.entities.Contact;

public class DtoContact {

	
	public static Contact getContact(int idContact) {
		
		return DaoContact.getContact(idContact);
	}
	
	public static List<Contact> getListContact() {
		
		return DaoContact.getListContact();
	}
}
